_config.yml

```diff
@@ -3,7 +3,7 @@ title: AJ Blogs
 subtitle: Blog page theme
 email: youremailaddress@gmail.com
 description: A simple and elegant jekyll theme for blogs.
-baseurl: "/aj-jekyll-blog-theme"
+baseurl: "/wsbctechblog"
```

Comment:
You have changed the `baseurl`. If you want make a blog for your website using this theme, then fork the repo and make changes in that repo.